samples/bluetooth/peripheral_uart/CMakeLists.txt

```diff
@@ -5,10 +5,6 @@
 #
 cmake_minimum_required(VERSION 3.13.1)
 
-set(PM_STATIC_YML_FILE
```

Comment:
Just out of curiosity, did you want to remove PM_STATIC also for non-Thingy:53 cases (same for other non-Thingy specific examples)?